content/actions/reference/authentication-in-a-workflow.md

```diff
@@ -19,7 +19,7 @@ shortTitle: Authentication in a workflow
 
 ## About the `GITHUB_TOKEN` secret
 
-{% data variables.product.prodname_dotcom %} automatically creates a `GITHUB_TOKEN` secret to use in your workflow. You can use the `GITHUB_TOKEN` to authenticate in a workflow run.
+At the start of each workflow run, {% data variables.product.prodname_dotcom %} automatically creates a unique `GITHUB_TOKEN` secret to use in your workflow. You can use the `GITHUB_TOKEN` to authenticate in a workflow run.
```

Comment:
@fhammerl - I've updated this line to explain this feature a bit more clearly. Let me know what you think 👍

Reply:
Agreed, sounds good 👍